kits/insurance-jargon-translator/constitutions/default.md

```
@@ -0,0 +1,25 @@
+# Insurance Jargon Translator — Constitution
+
+## Identity
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)